### PR TITLE
Feature/page setup: With implementation of a restaurante web page

### PR DESCRIPTION
**Pull request by Diegodsha**
- Use webpack to manage and compile assets: images and stylesheets
- Set a webpack config file
- Use javascript partials to keep code organized
- Use npm to install packages

### CODE REVIEW

File: src/home.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import createFooter from './footer';
+
+function createHomePage() {
+  const body1 = document.querySelector('body');
+  // change body background img
```
Comment (afanimartin) [inline src/home.js]: - Remove all comments from your code. They make the codebase look ugly.

File: src/home.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import createFooter from './footer';
+
+function createHomePage() {
```
Comment (afanimartin) [inline src/home.js]: - Use ES6 arrow function syntax instead

File: src/contact.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+function createContactPage() {
```
Comment (afanimartin) [inline src/contact.js]: - Use ES6 syntax

File: src/footer.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+function createFooter() {
```
Comment (afanimartin) [inline src/footer.js]: - Convert to ES6 syntax

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import 'bootstrap';
+import './style.scss';
+import createHomePage from './home';
+import createMenuPage from './menu';
+import createContactPage from './contact';
+
+// Removes selected tab color and cleans main content
+function clean() {
```
Comment (afanimartin) [inline src/index.js]: - Use ES6 syntax

File: src/menu.js [hunk 6]
```diff
@@ -0,0 +1,235 @@
+import lasagna1 from './assets/spinach-lasagna.jpg';
+import lasagna2 from './assets/Best-Ever-Vegetarian-Lasagna.jpg';
+import lasagna3 from './assets/Classic-Meat-Lasagna-4.jpg';
+import pomodoro from './assets/Pomodoro-pasta.jpg';
+import puttanesca from './assets/spaghetti-puttanesca.jpg';
+
+function createMenuPage() {
```
Comment (afanimartin) [inline src/menu.js]: - Use ES6 syntax and remove comments

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work. Your app looks amazing.
### Well done 🌵 🌵 🌵 
- proper use of modules
- readme looks professional
- required packages added to package.json

### Changes to make 🙅🏿 🙅🏿 🙅🏿 
- remove comments from your codebase. Comments make the code look ugly
- use ES6 syntax to define functions
- add commands for building, installing node packages using npm, and for running locally

### Martin Afani
#### Slack: whiz25

File: dist/index.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                <a class="nav-link contact" href="#">Contact</a>
+              </li>
+            </ul>
+            <span class="navbar-text d-none d-sm-block">
+              We love Lasagna <i class="fa fa-heart"></i
+            ></span>
+          </div>
+        </div>
+      </nav>
+    </div>
```
Comment (rindrajosia) [inline dist/index.html:50]: [OPTIONAL] It **appends** HTML elements to the **< div >** with **id=content** when the page is loaded.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel

- Good use of ES6 syntax :+1: 

To improve :factory_worker: [REQUIRED]:

- [ ] As I can see, you added two necessary run common commands in the `package.json` inside 'scripts' but it will be nice if you add another one ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)): `"dev": "webpack --mode development"`

Nice to have:

- [ ] It **appends**` HTML` elements to the `<div> with id=content `when the page is loaded 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: dist/index.html (hunk 7 above)
Comment (Diegodsha) [reply dist/index.html:50]: Hello, I don't understand this optional point:[OPTIONAL] It appends HTML elements to the < div > with id=content when the page is loaded.

Comment (Diegodsha) [reply dist/index.html:50]: Also, I already have set the development mode is it really necessary to add a script?

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow. 💯 
  - App is functionsl and available online. 💯 
  - It has a stupendous design. 💯
  
- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**